packages/rx_bloc_cli/mason_templates/bricks/rx_bloc_base/__brick__/lib/feature_splash/di/splash_page_with_dependencies.dart

```diff
@@ -0,0 +1,35 @@
+{{> licence.dart }}
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter_rx_bloc/flutter_rx_bloc.dart';
+import 'package:provider/provider.dart';
+
+import '../blocs/splash_bloc.dart';
+import '../views/splash_page.dart';
+
+class SplashPageWithDependencies extends StatelessWidget {
+  const SplashPageWithDependencies({
+    this.redirectToLocation,
+    Key? key,
+  }) : super(key: key);
+
+  final String? redirectToLocation;
+
+  List<RxBlocProvider> get _blocs => [
+    RxBlocProvider<SplashBlocType>(
+      create: (context) => SplashBloc(
+        context.read(),
+        context.read(),
+        location: redirectToLocation,
```

Comment:
Can we pick a prop name that conveys more information .. for example redirectLocation or something similar, since just `location` does not mean much

Reply:
Done